NotifyMe/serializers.py

```diff
@@ -0,0 +1,86 @@
+# import serializers from rest_framework
+
+from rest_framework import serializers
+from .models.notification import Notification 
+from .models.subscription import Subscription
+from .models.subscriptionPlan import SubscriptionPlan
+from .models.user import User
+from django.utils import timezone
+from datetime import datetime
+from datetime import timedelta
+from .models.notificationType import NotificationType
+
+class UserSerializer(serializers.ModelSerializer):
+    subscription_plan = serializers.PrimaryKeyRelatedField(queryset=SubscriptionPlan.objects.all())
+
+    class Meta:
+        model = User
+        fields = ('id', 'email_id', 'first_name', 'last_name', 'subscription_plan', 'created_at', 'updated_at')
+        depth=1
+    
+    # THIS IS HELPING IN CREATING SUBSCRIPTION INSTANCE FROM THE INSIDE OF USER... 
+    def create(self, validated_data):
+        subscription_plan = validated_data.pop('subscription_plan')
+        user = User.objects.create(subscription_plan=subscription_plan, **validated_data)
+        
+        start_date = timezone.now()
+        
+        if subscription_plan.subscription_plan == "Basic": # start_Date+30days
```

Comment:
create a constant.py there declare all the subscription_plan dont compare with hard coded values